### PR TITLE
Added build script for real_aes

### PR DESCRIPTION
**Pull request by saraswati-patra**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: r/real_aes/real_aes_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  rm -rf $CORE_PACKAGE_NAME
+  echo "$CORE_PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"
+
+fi
+
+if ! git clone $CORE_PACKAGE_URL $CORE_PACKAGE_NAME; then
+        echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+                echo "$CORE_PACKAGE_URL $CORE_PACKAGE_NAME"
+        echo "$CORE_PACKAGE_NAME  |  $CORE_PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+        exit 0
```
Comment (kumarvikas2020) [inline r/real_aes/real_aes_ubi_8.5.sh:44]: Exit code should be 1

File: r/real_aes/real_aes_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  rm -rf $PACKAGE_NAME
+  echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"
+
+fi
+
+ if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+        echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+                echo "$PACKAGE_URL $PACKAGE_NAME"
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+        exit 0
```
Comment (kumarvikas2020) [inline r/real_aes/real_aes_ubi_8.5.sh:74]: Exit code should be 1

**Review by kumarvikas2020 — CHANGES_REQUESTED** [review]

File: r/real_aes/real_aes_ubi_8.5.sh (hunk 1 above)
Comment (saraswati-patra) [reply r/real_aes/real_aes_ubi_8.5.sh:44]: code changes with exit code 1

File: r/real_aes/real_aes_ubi_8.5.sh (hunk 2 above)
Comment (saraswati-patra) [reply r/real_aes/real_aes_ubi_8.5.sh:74]: code changes with exit code 1

**Review by kumarvikas2020 — APPROVED** [review]

**Comment by kumarvikas2020** [comment]
@kandarpamalipeddi LGTM

**Pull request merged**